namespace/executor.go

```diff
@@ -0,0 +1,81 @@
+package namespace
+
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/longhorn/go-common-libs/types"
+	"github.com/longhorn/go-common-libs/utils"
+)
+
+// Executor is a struct resonpsible for executing commands in a specific
+// namespace using nsenter.
+type Executor struct {
```

Comment:
Just realized `Executor`'s methods are different than the interface ` ExecuteInterface`. Is it able to fit the interface?

Reply:
ExecuteInterface is used for general command execution. It is integrated into the Executor struct within the namespace package to wrap nsenter around the executed command. And 
 this is primarily for the purpose of unit-test mocking.

Reply:
Got it. Thanks.